content/open-source/melange/tutorials/getting-started-with-melange.md

```diff
@@ -378,12 +379,12 @@ docker load < hello-minicli.tar
 ```
 ```
 10f951ac3cd2: Loading layer [==================================================>]  7.764MB/7.764MB
-Loaded image: hello-minicli:test
+Loaded image: hello-minicli:test-%host-architecture%
 ```
 Now you can run your Minicli program with:
 
 ```shell
-docker run --rm hello-minicli:test
+docker run --rm hello-minicli:test-%host-architecture%
```

Comment:
this command didn't work for me as written, is it supposed to? If not, it could use a brief explainer to highlight like "Hey, use the image name returned in the output from the previous command here"

Reply:
That's a good idea if someone doesn't use the variable substitution thing to edit the command, will add!